app/src/main/java/swati4star/createpdf/util/FileUtils.java

```diff
@@ -108,7 +110,6 @@ private void shareFile(ArrayList<Uri> uris) {
      * opens a file in appropriate application
      *
      * @param path - path of the file to be opened
-     *
      */
```

Comment:
[I repeat] Any beautification work should be done in separate commit may or may not in single pull request.

Reply:
+1